pageserver/src/basebackup.rs

```diff
@@ -341,12 +346,17 @@ where
                     .await
                     .map_err(|e| BasebackupError::Server(e.into()))?;
 
+                if blocks.len() > self.timeline.conf.lazy_slru_download_threshold {
```

Comment:
`blocks` contains the blocks of just the current partition, `part`. That doesn't seem right.

Reply:
Thank you
I had wrong assumption that partition=SLRU kind.
Fixed